docs/core/entity/media-player.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Media Player Entity
-sidebar_label: Media Player
+title: Media Device Entity
```

Comment:
It is media player, not media device. Device are a different construct in Home Assistant.